libminifi/test/TestBase.h

```diff
@@ -251,6 +251,8 @@ class TestPlan {
   std::shared_ptr<core::Processor> addProcessor(const std::string &processor_name, utils::Identifier& uuid, const std::string &name, const std::initializer_list<core::Relationship>& relationships,
                                                 bool linkToPrevious = false);
 
+  std::shared_ptr<minifi::Connection> addConnection(const std::shared_ptr<core::Processor> source_proc, const core::Relationship& source_relationship, const std::shared_ptr<core::Processor>& destination_proc);
```

Comment:
I don't think the relationship being in the middle is the best signature, but this is only my personal opinion, I can live with this in case the majority prefers this.

Reply:
Well, this reflects the timeline of events.